src/tokens.ts

```diff
@@ -195,6 +198,7 @@ export const TOKEN_SYMBOLS_MAP = {
       [CHAIN_IDs.MUMBAI]: "0xe6b8a5CF854791412c1f6EFC7CAf629f5Df1c747",
       [CHAIN_IDs.LINEA_GOERLI]: "0xB4257F31750961C8e536f5cfCBb3079437700416",
       [CHAIN_IDs.OPTIMISM_SEPOLIA]: "0x9552a0a6624A23B848060AE5901659CDDa1f83f8",
+      [CHAIN_IDs.MODE]: "0xd988097fb8612cc24eeC14542bC03424c656005f",
```

Comment:
Similar to above

Reply:
For both of them, I do not believe so. Their testnet bridge has no option for bridging USDC, so that makes me think that there is no official deployment of a testnet token.